docs/pages/flexbox.md

```diff
@@ -163,6 +163,40 @@ To align an individual child, use the below classes. They use the same alignment
 </div>
 ```
 
+## Vanilla Flexbox Helper Classes
+
+Foundation also includes some helper classes for quickly applying flex
+container & direction attributes to elements.
+
+To make something a flex container, simply apply
+- `.flex-container`
+
+And to change its flex direction from row to column you can use the helper classes:
+
+- `.flex-direct-row` (default)
+- `.flex-direct-row-reverse`
+- `.flex-direct-column`
+- `.flex-direct-column-reverse`
```

Comment:
Can we call it flex-dir-row? Dir is more shorthand direct and dont want to type out direction 😜